.travis.yml

```diff
@@ -6,5 +6,5 @@ before_install:
 
 rvm:
   - 2.7
-  - 2.6
-  - 2.5
+  - 3.0
```

Comment:
@peterhaochen47  I think in pipelines we still have 2.5 in use ? 
if yes I would ask here to have 2.5 in addtion to this list.